js/app.js

```diff
@@ -1,58 +1,62 @@
-function Project(project){
-  for(key in project){
-    this[key] = project[key];
+'use strict';
+(function(module){
+  function Project(project){
+    for(var key in project){
+      this[key] = project[key];
+    }
   }
 
-}
+  Project.all = [];
 
-Project.all = [];
+  Project.prototype.toHtml = function(){
+    var $source = $('#project-template').html();
+    var template = Handlebars.compile($source);
+    return template(this);
+  };
 
-Project.prototype.toHtml = function(){
-  var $source = $('#project-template').html();
-  var template = Handlebars.compile($source);
-  return template(this);
-};
+  Project.loadAll = function(data){
+    data.forEach(function(ele) {
+      Project.all.push(new Project(ele));
+    });
+  };
 
-Project.loadAll = function(data){
-  data.forEach(function(ele) {
-    Project.all.push(new Project(ele));
-  });
-  Project.all.forEach(function(project){
-    $('#projects').append(project.toHtml());
-  });
-};
+  Project.totalLines = function(){
+    return Project.all.map(function(project){
+      return project.code.reduce(function(a, b){
```

Comment:
Yeah planning to expand upon this at some point soon, maybe adding stats per project, which could be used to refactor this function a bit.